scss/mixins/_pagination.scss

```diff
@@ -11,17 +11,63 @@
 
 // scss-docs-start pagination-max-items-mixin
 // Boosted mod
-// Ensure we only display `n` items:
-// - the first `n/2` items
-// - the last `n/2` items
-// - including the .active one (so maybe not the `n/2`th last item)
+// The pagination list has (n) items and is built in 3 parts:
+// - the begining of the list is made of the first (n/2) items
+// - the end of the list is made of the first (n/2) items
+// - the last part of the list is made of the active element (so maybe not the (n/2)th last item)
 @mixin pagination-max-items($pagination-max-items) {
-  display: none;
+  $half-items: $pagination-max-items * .5;
 
-  &.active,
-  &:nth-child(-n+#{$pagination-max-items * .5}),
-  &:nth-last-child(-n+#{$pagination-max-items * .5}) {
-    display: list-item;
+  // rule 0:
+  //  - adding the (n/2) first items
+  //  - adding the (n/2 - 1) last items
+  //  - adding the (n/2) last item if the list contains exactly (n) items
```

Comment:
And if the list contains doesn't exactly (n) items?

Reply:
We add some '...' as an ellipsis for items that can't be displayed.